src/core/manager.c

```diff
@@ -126,6 +127,107 @@ static int manager_run_environment_generators(Manager *m);
 static int manager_run_generators(Manager *m);
 static void manager_vacuum(Manager *m);
 
+int manager_spawn_workers(Manager *m) {
+        char socket_fd_number[DECIMAL_STR_MAX(int) + 1];
+        _cleanup_free_ char *log_level = NULL;
+        static int n_workers_cache = -2;
```

Comment:
yuck.

at least define an anonymous enum with recognizable names for these special case